custom/icds_reports/views.py

```diff
@@ -1773,7 +1773,7 @@ def valid_state_names(self):
 
 
 @location_safe
-@method_decorator([api_auth, csrf_exempt, toggles.ICDS_NIC_INDICATOR_API.required_decorator()], name='dispatch')
```

Comment:
surely this should still be `csrf_exempt`

Reply:
for get request also? is not it only required in POST? I had to put it just because of POST